### PR TITLE
PWGGA/GammaConv: added GammaMCStudiesTask & more

### PR DESCRIPTION
**Pull request by fjonasALICE**
- added GammaMCStudies task allowing to study various photon properties on generator level. Please not that this task will only work, once changes in AliRoot get approved
- fixed a bug in heavy neutral meson task: in the past, fixed pi0 masses were not correctly propagated to validated MC level. These histograms are now added and should be taken for the respective efficiency calculations
- implemented proper fixing of charged pion masses

### CODE REVIEW

File: PWGGA/GammaConv/AliPrimaryPionCuts.cxx [hunk 1]
```diff
@@ -114,7 +114,7 @@ AliPrimaryPionCuts::AliPrimaryPionCuts(const char *name,const char *title) : Ali
 	fCutString=new TObjString((GetCutNumber()).Data());
 
 	// Using standard function for setting Cuts
-	Bool_t selectPrimaries=kFALSE;
+	Bool_t selectPrimaries=kTRUE;
```
Comment (fjonasALICE) [inline PWGGA/GammaConv/AliPrimaryPionCuts.cxx]: sorry this change was not intended to be in here, will fix before approval

**Comment by dmuhlhei** [comment]
+1

**Comment by fjonasALICE** [comment]
Now that I think about it, it is obvious that not only my task isn't usable yet, it will not build against AliRoot. I will remove the commit in question until my AliRoot request is approved. Sorry for the inconveniences

**Comment by nschmidtALICE** [comment]
+test

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**